File: src/main/java/org/example/kotlinWorkshop/java/lesson02/DAWhile.java
package org.example.kotlinWorkshop.java.lesson02;

public class DAWhile {

  public void whileLoop() {
    int x = 5;
    while (x > 0) {
      System.out.println(x);
      x--;
    }
  }


  public void doWhileLoop() {
    int x = 5;
    do {
      System.out.println(x);
      x--;
    } while (x > 0);
  }
}

File: src/main/kotlin/org/example/kotlinWorkshop/kotlin/lesson02/DAWhile.kt
package org.example.kotlinWorkshop.kotlin.lesson02

class DAWhile {
    fun whileLoop() {
        var x = 5
        while (x > 0) {
            println(x)
            x--
        }
    }


    fun doWhileLoop() {
        var x = 5
        do {
            println(x)
            x--
        } while (x > 0)
    }
}